rte/kernels-openacc/mo_rte_solver_kernels.F90

```diff
@@ -1238,6 +1266,7 @@ subroutine lw_solver_1rescl(ncol, nlay, ngpt, top_at_1, D,
                                              trans       ! transmissivity  = exp(-tau)
     real(wp), dimension(ncol,nlay,ngpt) :: source_dn, source_up
     real(wp), dimension(ncol,     ngpt) :: source_sfc, sfc_albedo
+    real(wp), dimension(ncol,nlay+1,ngpt) :: dummy
```

Comment:
What is there a dummy variable? Seems like we could delete it with no problem

Reply:
That is a problem I am facing. Since lw_transport_noscat requires non optional radn_upJac. I created a dummy variable. For the sake of clarity, I renamed it to dummyRadn_upJac